kombu/transport/redis_cluster.py

```diff
@@ -161,6 +161,10 @@ def _register(self, channel, client, conn, cmd):
 
     def _unregister(self, channel, client, conn, cmd):
         sock = self._chan_to_sock[(channel, client, conn, cmd)]
+
+        del self._fd_to_chan[sock.fileno()]
+        del self._chan_to_sock[(channel, client, conn, cmd)]
```

Comment:
Is it for preventing MovedError exception as you noted in the PR description?

Reply:
It's just cleaning up